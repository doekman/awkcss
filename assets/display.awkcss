BEGIN {
	width(30);
}
/^#/ {
	display(none);
}
