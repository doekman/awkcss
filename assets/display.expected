This line is shown            
This line is shown again      
So is this one                
